services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java

```diff
@@ -208,9 +208,9 @@ private boolean isEfgsEnforceDateBasedDownloadAndNotSeen(String batchTag) {
           batchContainsInvalidKeys.set(true);
           logger.info("{} {} keys failed validation and were skipped", batchInfo.getSourceSystem(), numOfInvalidKeys);
         }
-        int insertedKeys = diagnosisKeyService.saveDiagnosisKeys(validDiagnosisKeys);
-        logger.info("Successfully inserted {} {} keys for date {} and batchTag {}",  batchInfo.getSourceSystem(),
-            insertedKeys, date, batchTag);
+        diagnosisKeyService.saveDiagnosisKeys(validDiagnosisKeys);
+        logger.info("Successfully inserted {} keys for date {} and batchTag {}", batchInfo.getSourceSystem(),
+            date, batchTag);
```

Comment:
@pithumke 
that's going to be an issue... when we can't tell from the logs anymore how many keys did came in from which federation service